projects/projects.md

```diff
@@ -1,18 +1,22 @@
 ---
-title: "Projects"
-author: "Benedikt Hegner"
+title: "HSF Affiliate Projects"
+author: Benedikt Hegner, Graeme Stewart
 layout: default
+redirect_from: /projects.html
 ---
 
-# HSF Projects
+# HSF Affiliate Projects
 
-HSF seeks to serve new and emerging common projects through a project incubator activity. Templates to guide and aid new projects are being established. If you'd like your project to be involved just let us know. Talk to any member of the HSF Steering Group or [email](mailto:hsf-steering@googlegroups.com).
+This is the list of projects which have become [HSF Affiliated]({{site.baseurl}}/projects/affiliated.html).
 
-The incubator idea is close to the one of the Apache Software Foundation. Brian Van Klaveren who gave the exceptional ASF talk at the SLAC workshop provides [this link](http://www.apache.org/foundation/how-it-works.html#incubator) describing how ASF Incubator projects work.
+{:.table .table-hover .table-condensed .table-striped}
+| Name | Description | Affiliation | Year |
+| --------| ------------- |----------|-----------|
+| [PRmon](https://github.com/HSF/prmon) | Standalone lightweight process resource consumption monitor | ![Bronze](/images/HSF-logo/HSF-Bronze_Project-CD7F32.svg) | 2024 |
```

Comment:
Is PRmon the official way to write it, not prmon? OK :).

Reply:
My bad, it really is "prmon". Apparently I don't know my own projects 🫢